README.md

```diff
@@ -10,3 +10,30 @@ CodeFlare Stack Compatibility Matrix
 | CodeFlare-SDK                | v0.4.4  |
 | InstaScale                   | v0.0.4  |
 | KubeRay                      | v0.5.0  |
+
+## Release process
+
+Prerequisite:
+- Build and release [MCAD](https://github.com/project-codeflare/multi-cluster-app-dispatcher)
+- Build and release [InstaScale](https://github.com/project-codeflare/instascale)
+- Build and release [CodeFlare-SDK](https://github.com/project-codeflare/codeflare-sdk)
+
```

Comment:
Well, I haven't found any direct dependency on MCAD or InstaScale code apart from an image declaration in https://github.com/project-codeflare/codeflare-operator/blob/main/controllers/defaults.go .

I mentioned these repositories as prerequisites because I consider operator as a "main" repo, handling other components, so to build it as last.

Reply:
https://github.com/project-codeflare/codeflare-operator/blob/31806041e8a977fda1a4d300460a47237a5558c7/Makefile#L13

This value needs to be updated and the new defaults file must be generated if we are targeting a new release of InstaScale with the new CodeFlare operator release. The MCAD_VERSION variable will also need to be updated once @astefanutti 's PR with an MCAD version/custom image support is merged.